dxr/plugins/descriptor/__init__.py

```diff
@@ -48,9 +48,24 @@ class FileToIndex(dxr.indexers.FileToIndex):
     """Do lots of work to yield a description needle."""
 
     comment_re = re.compile(r'(?:.*?/\*+)(?:\s*\*?\s*)(?P<description>.*?)(?:(?:\*+/.*)|(?:$))', flags=re.M)
-    docstring_re = re.compile(r'(?:\'\'\'|""")(?:\s*)(?P<description>.*?)(?:(?:(\'\'\'|"""))|(?:$))', flags=re.M)
+    docstring_re1 = re.compile(r'"""\s*(?P<description>[^"]*)', flags=re.M)
```

Comment:
Why not fold these into 1 regex with a | between them, simplifying the code below?